releases/net472/KnownIssues/560016-XBAP Applications targeting .NET 2.0,3.5 select wrong runtime version.md

```diff
@@ -0,0 +1,28 @@
+# [|Known Issue Title|]
```

Comment:
The title (which appears on line 2) should go here instead of [[Known Issue Title|].

Reply:
Fixed